manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -109,22 +101,18 @@ spec:
         severity: warning
     - alert: etcdBackendQuotaLowSpace
       annotations:
-        summary: etcd cluster database size exceeds the defined quota.
-        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the
-          defined quota on etcd instance {{ $labels.instance }}, please defrag or
-          increase the quota as the writes to etcd will be disabled when it is full.'
+        description: 'etcd cluster "{{ $labels.job }}": database size exceeds the defined quota on etcd instance {{ $labels.instance }}, please defrag or increase the quota as the writes to etcd will be disabled when it is full.'
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdBackendQuotaLowSpace.md
+        summary: etcd cluster database is running full.
```

Comment:
"running full" is not very specific about what is full.  Maybe rephrase to "etcd cluster database size is near quota" or something?